jsonld.js/src/test/scala/org/w3/banana/jsonldjs/io/JsonLdJsParserTest.scala

```diff
@@ -0,0 +1,39 @@
+package org.w3.banana
+package jsonldjs
+package io
+
+import zcheck.SpecLite
+import scalajs.concurrent.JSExecutionContext.Implicits.runNow
+import org.w3.banana.plantain._
+
+object JsonLdJsParserTest extends SpecLite {
```

Comment:
I have just ported SerialisationTestSuite to zcheck before realising a writer instance was needed, which we don't have...